### ISSUE TITLE
Large error message on startup from PlatformJs

### ISSUE THREAD

**Issue by edmundnoble** [japgolly/scalacss#89]
`PlatformJs` always calls `eval` on the source of platform.js prior to accessing the global `platform` variable. Many content security policies prohibit `eval`, including those of Chrome extensions, so this is causing a large error to be printed on startup by scalacss in the Chrome extension I'm working on. One work-around might be to make `PlatformJs` try accessing the global `platform` first, just in case platform.js has already run, so that I can take care of running it externally. A more invasive alternative might be to add the source of platform.js (via an SBT plugin or something) into the app's compiled JS to avoid using `eval` at all.

**Comment by japgolly** [comment]
It turns out that using PlatformJS at all is just overkill and doesn't add anything except an increase in JS size. On master, it has been removed as a default meaning that if you don't explicitly ask for it, Scala.JS will completely remove it from the output JS.

TL;DR: Fixed on master. Available in next version.
